pom.xml

```diff
@@ -18,18 +18,32 @@
     </properties>
 
     <dependencies>
-        <dependency>
-            <groupId>org.jboss.spec</groupId>
-            <artifactId>jboss-javaee-all-7.0</artifactId>
-            <version>1.0.3.Final</version>
-            <scope>provided</scope>
-        </dependency>
         <dependency>
             <groupId>org.jboss.ejb3</groupId>
             <artifactId>jboss-ejb3-ext-api</artifactId>
             <version>2.2.0.Final</version>
             <scope>provided</scope>
         </dependency>
+        <!-- https://mvnrepository.com/artifact/jakarta.ejb/jakarta.ejb-api -->
+        <dependency>
+            <groupId>jakarta.ejb</groupId>
+            <artifactId>jakarta.ejb-api</artifactId>
+            <version>4.0.1</version>
```

Comment:
Thanks very much for these changes, Diana! Just one small thing, we can add a dependency on the WildFly BOM and then we don't need to specify the versions here and below. See https://github.com/wildfly-security-incubator/elytron-examples/pull/135

Reply:
@fjuma good point! Fixed